src/sentry/integrations/gitlab/webhooks.py

```diff
@@ -44,6 +49,31 @@ def get_repo(self, integration, organization, event):
             return None
         return repo
 
+    def update_repo_data(self, repo, event):
+        """
+        Given a webhook payload, update stored repo data if needed.
+
+        Assumes a 'project' key in event payload, with certain subkeys. Rework
+        this if that stops being a safe assumption.
+        """
+
+        project = event["project"]
+
+        name_from_event = "{} / {}".format(project["namespace"], project["name"])
```

Comment:
```suggestion
        name_from_event = u"{} / {}".format(project["namespace"], project["name"])
```
Implementing Mark's previous suggestion.